AprilSylph/Palettes-for-Tumblr
# Add new color variables to built-in native palettes

### Description


The new Tumblr design system has a new (huge) set of theme color variables, which Palettes for Tumblr doesn't currently override (see https://github.com/AprilSylph/Palettes-for-Tumblr/issues/191). This copies the current set of variables into our native palette data (useful regex replace: `--(.*): (.*);` > `"$1": "$2",`).

### Testing steps


Select one of the built-in tumblr palettes that differs from the currently applied palette and confirm that elements such as the create button and notification badges are themed correctly. Confirm that elements are themed correctly on recently created pages that use the new theme color variables such as:

- https://www.tumblr.com/search/xkit/communities
- https://www.tumblr.com/communities/test-community-46174/settings
- https://www.tumblr.com/communities/communities-feedback/about

One can press shift-P repeatedly to cycle the selected Tumblr palette; in an ideal world this would have no visual effect on any of these pages when an updated extension palette is selected. This isn't actually the case, since some components (premium perks button, a selected community in the sidebar, the text below the new post bar, etc) are coded with DOM changes based on the currently selected palette, but there should be fewer of these changes with this PR.
